marklogic-data-hub-central/ui/src/components/flows/flows.tsx

```diff
@@ -891,30 +914,29 @@ const Flows: React.FC<Props> = ({
         formData.append("files", file);
       });
 
-      if (singleIngest) {
```

Comment:
Looks good now but appears like one of my bugfixes were undone with some conflict resolution here. This previous code is needed to fix the issue with running a single load step.